config/config.go

```diff
@@ -15,13 +15,25 @@ type (
 		SessionPath string `toml:"session_path"`
 		ScriptPath  string `toml:"script_path"`
 	}
+	IncludePath struct {
+		Path string `toml:"path"`
+	}
 	Config struct {
-		StartupScripts       []Script `toml:"startup_scripts"`
-		DefaultStartupScript string   `toml:"default_startup_script"`
+		IncludedPaths        []IncludePath `toml:"included_paths"`
```

Comment:
This would be updated to reflect a more descriptive collection of extended / additional configs (whatever it's changed to from the previous comment).